util.js

```diff
@@ -44,19 +44,21 @@ exports.wait = function () {
 }
 
 const originalValue = exports.originalValue = function (value) {
-  // setting `oldProps` for backward-compatibility with old metadta
-  const oldProps = ['cyphertext', 'private', 'unbox']
-
-  const metaProps = oldProps.concat('meta')
-  const original = value.meta && value.meta.original || {}
-
-  var o = {}
-  for (var key in value) {
-    if (!metaProps.includes(key))
-      o[key] = original[key] || value[key]
+  if (value.meta) {
```

Comment:
I think here how you are mutating and before it was copying will break stuff, because I believe these objects are cached, so modifying them will change the cached value. A subsequent request, that asks for the mapped data will get it unmapped because of this. (unless it's read for disk again - which will be quite hard to debug!)

Reply:
it's also essential the order of the keys are preserved, which is why it uses a for each in the previous version.

Reply:
Nice catch, thanks. I didn't consider that.

Reply:
@arj03 fixed that problem the first time :wink: